### DISCUSSION TITLE
Ecommerce elements fro Chakra UI Pro

### DISCUSSION THREAD

**Discussion by casta42** [chakra-ui/chakra-ui#9315] [Q&A] [answered]
Thanks for all the hard work! V3 looks amazing, just wondering if all the ecommerce templates/blocks from Chakra Pro V2 will be available in Chakra ProV3?

**Comment by segunadebayo** [comment, marked as answer]
Yes, we're working on migrating the eCommerce blocks as well. Should be live by next month or so.
